### DISCUSSION TITLE
Problem AI-Deck: Flashing WiFi example stops at 99%

### DISCUSSION THREAD

**Discussion by tahaneG** [bitcraze/discussions#484] [Q&A]
Hello everybody,
I have problems to get started with the AI-Deck. I am trying to flash the WiFi example on the AI-Deck, just as it is mentioned in the "getting-started-with-aideck" guide. But the flashing process stops at 99% everytime. I know that there was a bug with the firmware version from 2022.09. But the error also occurs with the new 2022.12. version.

Does anyone know what could cause this problem? I am using the following components/system:
- OS: Ubuntu 20.04.5 LTS
- Crazyflie: 2.1 rev B1
- AI-Deck: 1.1 rev C1

Thanks in advance!

**Comment by knmcguire** [comment]
Hi!

It seems that you have the older version of the AIdeck that requires you to flash the bootloader. This is not necessary for rev D's to be updated, but the Rev C's do need to be reflashed.  Do please double check underneath your AIdeck, closely above 'ETH zurich'. as some of the bags it came in were faulty. 

If you have flashed the bootloader already, it might be needed to do that again perhaps? https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-the-aideck-1-0-or-aideck-1-1-rev-a-c-
